packages/firebase_ai/firebase_ai/lib/src/api.dart

```diff
@@ -469,8 +469,8 @@ enum BlockReason {
 
   const BlockReason(this._jsonString);
 
-  // ignore: unused_element
-  static BlockReason _parseValue(String jsonObject) {
+  // ignore: public_member_api_docs
+  static BlockReason parseValue(String jsonObject) {
```

Comment:
Any reason we don't document this when `parseCitationMetadata` is properly documented?

Reply:
Also since we don't want to expose those function to public but we want to be able to test the function, we should use the `@visibleForTesting` https://api.flutter.dev/flutter/meta/visibleForTesting-constant.html